packages/runtime/container-runtime/src/inboundBatchAggregator.ts

```diff
@@ -82,55 +40,30 @@ class ScheduleManagerCore {
 		private readonly getClientId: () => string | undefined,
 		private readonly logger: ITelemetryLoggerExt,
 	) {
-		// Listen for delta manager sends and add batch metadata to messages
-		this.deltaManager.on("prepareSend", (messages: IDocumentMessage[]) => {
-			if (messages.length === 0) {
-				return;
-			}
-
-			// First message will have the batch flag set to true if doing a batched send
-			const firstMessageMetadata = messages[0].metadata as IRuntimeMessageMetadata;
-			if (!firstMessageMetadata?.batch) {
-				return;
-			}
-
-			// If the batch contains only a single op, clear the batch flag.
-			if (messages.length === 1) {
-				delete firstMessageMetadata.batch;
-				return;
-			}
-
-			// Set the batch flag to false on the last message to indicate the end of the send batch
-			const lastMessage = messages[messages.length - 1];
-			// TODO: It's not clear if this shallow clone is required, as opposed to just setting "batch" to false.
-
-			lastMessage.metadata = { ...(lastMessage.metadata as any), batch: false };
-		});
-
 		// Listen for updates and peek at the inbound
-		this.deltaManager.inbound.on("push", (message: ISequencedDocumentMessage) => {
-			this.trackPending(message);
-		});
+		this.deltaManager.inbound.on("push", this.trackPending);
 
-		// Start with baseline - empty inbound queue.
-		assert(!this.localPaused, 0x293 /* "initial state" */);
+		// We are intentionally directly listening to the "op" to inspect system ops as well.
+		// If we do not observe system ops, we are likely to hit an error when system ops
+		// precedes start of incomplete batch.
+		this.deltaManager.on("op", this.afterOpProcessing);
```

Comment:
Why move this and delete assert 0x293?

Reply:
That assert is unncessary? `localPaused` starts as false and the code before the assert doesn't change it.

Does the order matter? The other two pieces of code is setting up event handlers. I guess it could matter. `trackPending` may resumt the inbound queue which will call the "op" event handler and queue in a call to `afterOpProcessing`.

Reply:
Reverted the order back.